cas-server-integration-hazelcast/src/main/java/org/jasig/cas/ticket/registry/HazelcastTicketRegistry.java

```diff
@@ -122,7 +124,7 @@ private void addTicket(final Ticket ticket, final long ttl) {
      */
     @Override
     public Ticket getTicket(final String ticketId) {
-        return this.registry.get(ticketId);
+        return getProxiedTicketInstance(this.registry.get(ticketId));
```

Comment:
Changing this makes it unnecessary to add updateTicket method to public registry API.